pages/linux/setxkbmap.md

```diff
@@ -6,6 +6,22 @@
 
 `setxkbmap {{fr}}`
 
+- Set multiple keyboard layouts, their variants and switching option:
+
+`setxkbmap -layout us,de -variant ,qwerty -option 'grp:alt_caps_toggle'`
```

Comment:
We should wrap the bits that users can change in `{{` and `}}`.

Reply:
I added curly braces to all arguments that makes sense for user to change. But I was bit unsure if I need to set it to all changeable values or not. For example `grep :grp` or `grep {{:grp}}` in the last example.

Reply:
Cool, thanks!

Really, it's only the values that makes sense for the user to change in the contact of your command example.